### ISSUE TITLE
Can we have the option to set the header? 

### ISSUE THREAD

**Issue by harshmaur** [instantsearch/instantsearch-ion.rangeSlider#7]
Currently I need to add the header manually to the widget which is different from the way we add all other widgets in instantsearch. It would be nice to have it implemented in the plugin.

**Comment by redox** [comment]
Good idea 👍 And it shouldn't be too complex. I won't have time to focus on it in the next few days but I'll be happy to review any PR if you have more time 💯
